### ISSUE TITLE
bj_cree_east fails to compile with warnings

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyboards#45]
```
bj_cree_east.kmn: Warning: line 116  warning 209A: The rule will never be matched because its key code is never fired.
bj_cree_east.kmn: Warning: line 117  warning 209A: The rule will never be matched because its key code is never fired.
```

Temporarily allowing warnings on this keyboard so we can pass the build

**Comment by DavidLRowe** [comment]
When fixing this problem, perhaps you could also (a) delete the Thumbs.db file, and (b) in the bj_cree_east.kps file, include the year "2016" in the copyright line that currently has "copyright April" with no year.

**Comment by DavidLRowe** [comment]
Seems as though the copyright should be 1999-2016 for README.md, LICENSE.md and bj_cree_east.kmn, based on dates in history recorded in .kmn file.

Also copyright string in bj_cree_east.kmn has extra © symbols at the end.

The warnings are:
```
bj_cree_east.kmn: Warning: line 116  warning 209A: The rule will never be matched because its key code is never fired.
bj_cree_east.kmn: Warning: line 117  warning 209A: The rule will never be matched because its key code is never fired.
```
The lines in question seem to be:
```
    + any(ConsKey)    > index(CC,1)
    + any(CapConsKey) > index(CC,1)
```
which reference:
```
store(ConsKey) "ptkcmnlsÎyrfÿ"
store(CapConsKey) "PTKCMNLSÎYRFÿ"
```
@mcdurdin Is the error caused by the presence of characters in the store that are not on a keyboard (U+00CE and U+00FF, which occur in both stores)?

**Comment by mcdurdin** [comment]
@DavidLRowe, yes those keys can never be fired because they don't exist on US English. The message is possible a little inaccurate in this instance; some keys in the rule will never be fired, rather than the whole rule.

**Comment by DavidLRowe** [comment]
@mcdurdin Do we need to rewrite the .kmn file? relax the compiler a bit? leave the issue open and deal with it later?

**Comment by mcdurdin** [comment]
You can turn off the project option 'fail on warnings' for now. Long term we should resolve the issues in the file.

**Comment by LornaSIL** [comment]
Fixed by #387
